source/security/private-endpoints.txt

```diff
@@ -214,21 +188,161 @@ connection using either the App Services UI or the Admin API.
                  -H "Content-Type: application/json" \
                  -d '{
                    "cloud_provider_region": "<your-cloud-deployment-region>",
-                   "cloud_provider_endpoint_id": "<your-vpc-endpoint-id>"
+                   "cloud_provider_endpoint_id": "<your-vpc-endpoint-id>",
+                   "endpoint_type": "general",
+                   "comment": "<comment-about-endpoint>"
```

Comment:
Since users don't have the Admin API definition right here, should we make this placeholder more descriptive to make it easier for users to understand how to populate this? i.e.:
```suggestion
                   "comment": "Optional: add a comment describing the endpoint."
```